docs/getting-started.asciidoc

```diff
@@ -14,10 +14,14 @@ This tutorial shows you how to:
 
 Before you begin, make sure you've <<install, installed Kibana>> and established
 a {kibana-ref}/connect-to-elasticsearch.html[connection to Elasticsearch].
-
 You might also be interested in the
 https://www.elastic.co/webinars/getting-started-kibana[Getting Started with Kibana]
 video tutorial.
+
+If you are running our
+https://www.elastic.co/cloud/elasticsearch-service[hosted Elasticsearch Service]
+on Elastic Cloud, you can {cloud}/ec-enable-kibana.html[enable Kibana] with
+the flick of a switch.
```

Comment:
![image](https://user-images.githubusercontent.com/15148011/43016080-8318d710-8c06-11e8-854a-9396103aacaf.png)


Kibana is always enabled for new clusters and has been for a while (the screenshot above is of a side panel that pops up when you create a new deployment to show you what I mean), so I would recommend that we don't point people to the legacy enablement task for older clusters. It's unnecessary and in most cases simply doesn't apply. Also, the text is aimed at people _already_ running on the cloud, so it might make sense to point to the Elasticsearch Service console rather than the product overview page on elastic.co. (If users don't have an account trial, they can sign up for a free trial from there.)

With these suggestions in mind, I would change this text to:

"If you are running our https://cloud.elastic.co[hosted Elasticsearch Service] on Elastic Cloud, you can access Kibana with a single click."

Let me know if you'd like to chat about this further or if you think these suggestions misconstrue the intent of the notice.

Reply:
@gchaps can we update the install docs with this change too then? nice catch @nrichers !

Reply:
Will do